gqcpy/include/interfaces.hpp

```diff
@@ -699,4 +700,145 @@ void bindSimple2DMInterface(Class& py_class) {
 }
 
 
+/*
+ *  MARK: QCModel - Hartree-Fock
+ */
+
+/**
+ *  Add Python bindings for some APIs related to `QCModel::R/U/GHF`.
+ * 
+ *  @tparam Class               The type of the Pybind11 `class_` (generated by the compiler).
+ *  
+ *  @param py_class             The Pybind11 `class_` that should obtain APIs related to `QCModel::R/U/GHF`.
+ */
+template <typename Class>
+void bindQCModelHartreeFockInterface(Class& py_class) {
+
+    // The C++ type corresponding to the Python class.
+    using Type = typename Class::type;
+    using Hamiltonian = typename Type::Hamiltonian;
+    using Scalar = typename Type::Scalar;
+
+    py_class
+        .def(
+            "calculateOrthonormalBasis1DM",
+            [](const Type& parameters) {
+                return parameters.calculateOrthonormalBasis1DM();
+            },
```

Comment:
```suggestion
            &Type::calculateOrthonormalBasis1DM,
```

I think this implementation can become shorter.

Reply:
If written this way, I get the following error:

![Screenshot 2020-12-07 at 13 56 25](https://user-images.githubusercontent.com/54795591/101353515-03807e80-3894-11eb-833a-7259ea0f4c61.png)

I believe this may be the case because the `QCModel` contains a `public` and a `static` `calculateOrthonormalBasis1DM()` API.

Reply:
Yup, that's true! An oversight on my part.